src/views/textview/text-view.js

```diff
@@ -15,14 +15,43 @@ import styles from './text-view.styles';
 const TextViewInfoModalContent = () => html`
   <div>
     <p>
-      The color coding in the text in the left panel indicates how many
-      parallels are to be encountered at a certain position given the current
-      filters.
+      The color coding of the syllables in the Inquiry Text on the left side
+      indicates how many approximate matches are to be encountered at a certain
+      syllable according to the current filter settings.
     </p>
-    <p>
-      The colors range from light blue (1 parallel) to red (5 or more parallels)
-    </p>
-    <p>When no colors are displayed, please change the filter settings.</p>
+    <p><b>Color codes per number of matches:</b></p>
+    <table style="width:100%; table-layout:fixed" align="center">
+      <tr>
+        <td><b>0</b></td>
+        <td><b>1</b></td>
+        <td><b>2</b></td>
+        <td><b>3</b></td>
+        <td><b>4</b></td>
+        <td><b>5</b></td>
+        <td><b>6</b></td>
+        <td><b>7</b></td>
+        <td><b>8</b></td>
+        <td><b>9</b></td>
+        <td><b>10 or more</b></td>
+      </tr>
+      <tr>
+        <td bgcolor="#000000" style="height:30px"></td>
```

Comment:
can we get these colors from javascript here to avoid having to update them in two places?

Reply:
Agreed ... have changed this.